dev-util/directx-headers/directx-headers-1.606.3.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
`# Copyright 2022 Gentoo Authors` (which can become `# Copyright 2022-2023 Gentoo Authors` in next year).